src/TabsInternal.elm

```diff
@@ -0,0 +1,205 @@
+module TabsInternal exposing (Config, Tab, views)
```

Comment:
thinking in terms of atomic design, why aren't the things in this module an atom? Are they just harder to use correctly so we wrap them in preset ways?

Reply:
From my perspective, we wrap them in order to have our exposed components match the Designer's styleguide. I don't know that we'll want another tab component apart from SegmentedControls and Tabs? I suppose we may someday...